src/mapping/impl/CreateClustering.hpp

```diff
@@ -364,7 +367,9 @@ inline std::tuple<double, Vertices> createClustering(mesh::PtrMesh inMesh, mesh:
   // maximum distance between cluster centers lying diagonal to each other. The maximum distance takes the overlap condition into
   // account: if the distance between the centers is sqrt(2) * radius, we violate the overlap condition between diagonal clusters
   // 0.3 should be a good default value
-  const double maximumCenterDistance = std::sqrt(2) * clusterRadius * (1 - relativeOverlap);
+  // @todo: most of the 3D experiments ran with the default of 0.3 which would correspond to relativeOverlap of 0, i.e., the limit
+  // Can we use different values for 2D and 3D? Which value should we use?
+  const double maximumCenterDistance = inMesh->getDimensions() == 2 ? std::sqrt(2) * clusterRadius * (1 - relativeOverlap) : clusterRadius * (1 - relativeOverlap);
```

Comment:
Where could we document such things for users?

Be careful: Eventually, preCICE will also be able to handle 1D cases. Better safeguard with an assertion, such that we know that we need to update things here.

Reply:
Not sure how far this eventually is in the future, because there are many places where we or I assume to have at least two dimensions. I added an assert